src/runner/settings_window.cpp

```diff
@@ -494,6 +501,86 @@ void close_settings_window()
 void open_oobe_window()
 {
     std::thread([]() {
-        run_settings_window(true);
+        run_settings_window(true, std::nullopt);
     }).detach();
 }
+
+std::string ESettingsWindowNames_to_string(ESettingsWindowNames value)
+{
+    switch (value)
+    {
+    case ESettingsWindowNames::Overview:
+        return "Overview";
+    case ESettingsWindowNames::Awake:
+        return "Awake";
+    case ESettingsWindowNames::ColorPicker:
+        return "ColorPicker";
+    case ESettingsWindowNames::FancyZones:
+        return "FancyZones";
+    case ESettingsWindowNames::Run:
+        return "Run";
+    case ESettingsWindowNames::ImageResizer:
+        return "ImageResizer";
+    case ESettingsWindowNames::KBM:
+        return "KBM";
+    case ESettingsWindowNames::PowerRename:
+        return "PowerRename";
+    case ESettingsWindowNames::FileExplorer:
+        return "FileExplorer";
+    case ESettingsWindowNames::ShortcutGuide:
+        return "ShortcutGuide";
+    case ESettingsWindowNames::VideoConference:
+        return "VideoConference";
+    }
+    return "";
+}
+
+ESettingsWindowNames ESettingsWindowNames_from_string(std::string value)
+{
+    if (value.compare("Overview") == 0)
```

Comment:
nit: why not just use `value == "Overview"` etc.?

Reply:
Old habits :) Will do it